packages/docs/pages/Checkbox/CheckboxPage.tsx

```diff
@@ -25,7 +25,26 @@ export default () => (
         return (
           <Form>
             <FormGroup>
-              <Checkbox label={checked ? 'Checked' : 'Unchecked'} checked={checked} onChange={handleChange} />
+              <Checkbox
+                label={checked ? 'Checked' : 'Unchecked'}
+                checked={checked}
+                onChange={handleChange}
+                description={{
+                  text: 'Description for off.',
+                }}
+              />
+              <Checkbox
+                label="Disabled"
+                disabled={true}
+                description={{
+                  text: 'Description for off.',
+                  link: {
+                    text: 'Learn more',
+                    target: '_blank',
+                    href: 'http://www.bigcommerce.com',
+                  },
+                }}
+              />
```

Comment:
Should have caught this one too, but instead of adding `description` on the main example, could we create another example above/below the "Indeterminate" one?